change/@fluentui-react-charting-a75bd7cf-c554-4423-8550-af0b5c84e76b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Charting:Add New Sankey chart to charting package",
```

This should be minor since it's a new feature, and the note should be explicit that it will break IE 11.
```suggestion
  "type": "minor",
  "comment": "Add Sankey chart **(not compatible with IE 11)**",
```